contracts/epochs/FlowEpoch.cdc

```diff
@@ -857,7 +944,14 @@ pub contract FlowEpoch {
                                            numViewsInDKGPhase: numViewsInDKGPhase,
                                            numCollectorClusters: numCollectorClusters,
                                            FLOWsupplyIncreasePercentage: FLOWsupplyIncreasePercentage)
-        
+
+        // Set a reasonable default for the epoch timing config targeting 1 block per second
+        let defaultEpochTimingConfig = EpochTimingConfig(
+            duration: numViewsInEpoch,
+            refCounter: currentEpochCounter,
+            refTimestamp: UInt64(getCurrentBlock().timestamp))
```

Comment:
```suggestion
            refTimestamp: UInt64(getCurrentBlock().timestamp)+numViewsInEpoch)
```
Default reference timestamp should be at the _end_ of the first epoch. Currently it is at the beginning. This is my mistake from #395